### ISSUE TITLE
It's an interface without anything to access after climbing the wall. Can you please handle it.

### ISSUE THREAD

**Issue by wangshangyao** [archylex/OVPND#2]
It's an interface without anything to access after climbing the wall. Can you please handle it. Reply me if it's convenient, or can I send the file to my email? Wsy1767112006@gmail.com

**Comment by archylex** [comment]
> It's an interface without anything to access after climbing the wall. Can you please handle it. Reply me if it's convenient, or can I send the file to my email? [Wsy1767112006@gmail.com](mailto:Wsy1767112006@gmail.com)

Yeah, it's just parser of OpenVPN configuration files. You can install OpenVPN app from Google Play or something else. (OpenVPN for Android) and set this ovpn files.
